commcare_export/env.py

```diff
@@ -250,11 +250,44 @@ def str2date(val):
 
     return date.replace(microsecond=0, tzinfo=None)
 
-@unwrap
+@unwrap('val')
 def bool2int(val):
     return int(str2bool(val))
 
 
+@unwrap('val')
+def selected_at(val, index):
+    if not val:
+        return None
+
+    try:
+        index = int(index)
+    except ValueError:
+        return "Error: index must be an integer"
+
+    try:
+        return str(val).split(' ')[index]
```

Comment:
might be better to use .split() rather than split(' ') since .split() is more forgiving of contiguous whitespace, or leading or trailing whitespace. or does that not matter here because we always expect to be passed a multi-select value?